viewing_party/party.py

```diff
@@ -1,23 +1,153 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+
+    movie = {}
+    movie["title"] = title
+    movie["genre"] = genre
+    movie["rating"] = rating
+
+    return movie

```

Comment:
We can directly return a dictionary literal without creating `movie` first
```suggestion
    return {
	        "title": title, 
	        "genre": genre, 
	        "rating": rating 
	        } 
```